k8s/perftest/common/ccd/definition-store-api.yaml

```diff
@@ -30,6 +30,7 @@ spec:
         DEFINITION_STORE_DB_OPTIONS: "?sslmode=require&gssEncMode=disable"
         DEFINITION_STORE_DB_MAX_POOL_SIZE: 32
         DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-perftest.postgres.database.azure.com
+        DEFINITION_STORE_S2S_AUTHORISED_SERVICES: em_hrs_api
```

Comment:
this will break perftest env. We need to make sure we are appending the service id to the allowed-listed ids inherent from chart config.